skcosmo/preprocessing/_flexible_scaler.py

```diff
@@ -0,0 +1,600 @@
+import numpy as np
+from sklearn.base import (
+    BaseEstimator,
+    TransformerMixin,
+)
+from sklearn.preprocessing._data import KernelCenterer
+from sklearn.utils.validation import (
+    FLOAT_DTYPES,
+    _check_sample_weight,
+    check_is_fitted,
+)
+
+
+class StandardFlexibleScaler(TransformerMixin, BaseEstimator):
+    """Standardize features by removing the mean and scaling to unit variance.
+    Make the mean of the columns equal to zero and the
+    variance of each column (`column_wise==True`) equal to one.
+    The standard score of a sample `x` is calculated as:
+
+        z = (x - u) / s
+
+    where `u` is the mean of the training samples or zero if `with_mean=False`,
+    and `s` is the standard deviation of the training samples or one if
+    `with_std=False`.
```

Comment:
```suggestion
    where `u` is the mean of the samples if `with_mean`, otherwise zero,
    and `s` is the standard deviation of the samples if `with_std` or one.
```